src/client/app/containers/LineChartContainer.ts

```diff
@@ -81,7 +82,16 @@ function mapStateToProps(state: State) {
 						xData.push(timeReading.format('YYYY-MM-DD HH:mm:ss'));
 						const readingValue = reading.reading * scaling;
 						yData.push(readingValue);
-						hoverText.push(`<b> ${timeReading.format('ddd, ll LTS')} </b> <br> ${label}: ${readingValue.toPrecision(6)} ${unitLabel}`);
+						const minValue = reading.min * scaling;
+						yMinData.push(minValue);
+						const maxValue = reading.max * scaling;
+						yMaxData.push(maxValue);
+						if (state.graph.showMinMax) {
```

Comment:
As an optimization, the filling of these arrays should be inside of the if statement. If showMinMax is false, these arrays are not used, so it is wasteful to fill them with data.